tests/components/influxdb/test_init.py

```diff
@@ -397,16 +388,16 @@ async def test_event_listener(
         state = MagicMock(
             state=in_,
             domain="fake",
-            entity_id="fake.entity-id",
-            object_id="entity",
+            entity_id="fake.entity_id",
+            object_id="entity_id",
             attributes=attrs,
         )
-        event = MagicMock(data={"new_state": state}, time_fired=12345)
+        MagicMock(data={"new_state": state}, time_fired=12345)
```

Comment:
This line doesn't seem to have an effect?

Reply:
nope, we can get rid of this.

Reply:
34dd64582f2f5ab4d52aa7ca0e856758e56cc177